index.js

```diff
@@ -16,12 +16,15 @@ class Dynastar {
    *  @param {String} [options.hashKey='key'] The hashKey of the model
    *  @param {String} [options.rangeKey] The rangeKey for the model
    *  @param {Function} [options.createKey] The function to create the hashKey for certain models
+   *  @param {Function} [options.createHashKey] The function to create the hashKey for certain models (overrides createKey)
+   *  @param {Function} [options.createRangeKey] The function to create the rangeKey for certain models
    */
-  constructor({ model, hashKey = 'key', rangeKey, createKey, ...fns }) {
+  constructor({ model, hashKey = 'key', rangeKey, createKey, createHashKey, createRangeKey, ...fns }) {
     this.model = model;
     this.hashKey = hashKey;
     this.rangeKey = rangeKey;
-    this._createKey = createKey;
+    this._createKey = createHashKey || createKey;
```

Comment:
Ah, so this is just another name for `createKey`. I assume `createHashKey` is preferred. Are we going to deprecate `createKey` and remove it in the next major version?

Reply:
Looks like we never properly documented `createKey` in the `README.md`. Can you add documentation for these two new functions, maybe call out that `createHashKey` will be replacing `createKey`?